neuraxle/steps/misc.py

```diff
@@ -161,6 +161,23 @@ def inverse_transform(self, processed_outputs):
 
 
 class CallbackWrapper(HandleOnlyMixin, MetaStepMixin, BaseStep):
+    """
+    A step that calls a callback function for each of his methods : transform, fit, fit_transform, and even inverse_transform.
+    To be used with :class:`TapeCallbackFunction`.
```

Comment:
This class should go in the same file as all the Tape classes (if it's not already in the same file), as it seems that your CallbackWrapper is used only for testing as well and is strongly tied to those classes.

Reply:
already in the same file all good.